plugins/couchbase/src/main/java/org.radargun.service/CouchbaseClientOperations.java

```diff
@@ -0,0 +1,105 @@
+package org.radargun.service;
```

Comment:
I just saw that the folder structure is using (dot) instead of a folder separator.
The name of the file is: `org.radargun.service/CouchbaseClientOperations.java` and it should be `org/radargun/service/CouchbaseClientOperations.java`